### PR TITLE
add haproxy support for dashboard

### PR DESCRIPTION
**Pull request by avanthakkar**
Signed-off-by: Avan Thakkar <athakkar@redhat.com>

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -127,6 +127,12 @@ services:
             - KEYCLOAK_IMPORT=/docker/saml-demo-realm.json
             - BIND=127.0.0.1
         scale: -1
+    
+    haproxy:
+        image: haproxy
```
Comment (alfonsomthd) [inline docker-compose.yml]:
```suggestion
        image: ${HAPROXY_IMAGE:-haproxy:latest}
```

Comment (alfonsomthd) [reply docker-compose.yml]: For consistency, under ```image:```, it should be added:
```
        container_name: haproxy
        hostname: haproxy
```

Also, in ```.env.example``` it should be added:
HAPROXY_HOST_HTTP_PORT=80
HAPROXY_HOST_HTTPS_PORT=443
#HAPROXY_IMAGE=

**Review by alfonsomthd — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (avanthakkar) [reply docker-compose.yml]: Done!

**Comment by avanthakkar** [comment]
Closing it as it's already covered here https://github.com/rhcs-dashboard/ceph-dev/commit/2d20eb64fc1d619fcdff3d333754552cbdc1a6d3

**Pull request closed without merging**